src/vue/align-editor/align-editor.vue

```diff
@@ -1,33 +1,34 @@
 <template>
   <div class="alpheios-alignment-editor-container" v-show="showAlignEditor">
-      <h2>Define Alignment for Source and Translation Texts
+      <h2>Define Alignment for Origin and Target Texts

```

Comment:
although I know it's a pain, could we start with all text in locale files for ease of localization? We should be able to reuse the l10n library from alpheios-core.

Reply:
ok - will add

Reply:
big change